app/views/play/level/WebSurfaceView.coffee

```diff
@@ -35,6 +38,7 @@ module.exports = class WebSurfaceView extends CocoView
     html = _.find(dom, name: 'html') ? {name: 'html', attribs: null, children: [body]}
     # TODO: pull out the actual scripts, styles, and body/elements they are doing so we can merge them with our initial structure on the other side
     { virtualDom, styles, scripts } = @extractStylesAndScripts(@dekuify html)
+    @extractCssSelectors(styles, e.html)
```

Comment:
I'm a bit torn on this — It feels weird to not capitalize acronyms, but it's also weird to capitalize them and make it seems like the acronym is "CSSS". I've been following this convention as much as possible: http://stackoverflow.com/questions/15526107/acronyms-in-camelcase